frontend/src/RecipeList/RecipeList.jsx

```diff
@@ -38,7 +38,7 @@ const RecipeList = () =>{
     };
 
     const handleCloseModal = () => {
-        setModalOpen(modalOpen);
+        setModalOpen(!modalOpen);
```

Comment:
can you just put here `false`? Since close function should only be closing the modal?